x-pack/qa/rolling-upgrade/build.gradle

```diff
@@ -160,9 +163,17 @@ for (Version bwcVersion : bwcVersions.wireCompatible) {
         'mixed_cluster/80_transform_jobs_crud/Test put batch transform on mixed cluster',
         'mixed_cluster/80_transform_jobs_crud/Test GET, start, and stop old cluster batch transforms',
         'mixed_cluster/80_transform_jobs_crud/Test put continuous transform on mixed cluster',
-        'mixed_cluster/80_transform_jobs_crud/Test GET, stop, start, old continuous transforms'
+        'mixed_cluster/80_transform_jobs_crud/Test GET, stop, start, old continuous transforms',
+        //Cannot determine if we will throw a warning on get as we might hit an old node
+        //Cannot determine if we will have a warning on get _stats as we might hit an old node
+        'mixed_cluster/40_ml_datafeed_crud/Test old cluster datafeed with aggs'
       ])
     }
+    // Skip putting aggs if we have a mixed cluster were certain nodes will return a deprecation warning
+    // and others will not.
+    if (bwcVersion.before('7.2.0')) {
+      toBlackList << 'mixed_cluster/40_ml_datafeed_crud/Put job and datafeed with aggs in mixed cluster'
```

Comment:
Again, it is possible here that the `PUT` does not hit a new node at all and no deprecation warning is returned :(